modules/dhcp/dhcp_api.rb

```diff
@@ -102,15 +101,10 @@ def load_subnet_data
       load_subnet
       load_subnet_data
 
+      content_type :json
       record = server.find_record(@subnet.network, params[:record])
       log_halt 404, "Record #{params[:network]}/#{params[:record]} not found" unless record
       server.del_record @subnet, record
```

Comment:
I'm unsure if this is correct with the native_ms provider - it looks like `del_record` will return the string response from the `execute` call to delete the record. Both the libvirt and ISC providers look like they're going to return `nil`, but the execute response won't be valid JSON if that's what's returned.